packages/sdks/snippets/sveltekit/src/routes/[...catchall]/+page.server.js

```diff
@@ -4,7 +4,7 @@
  */
 import { fetchOneEntry, getBuilderSearchParams } from '@builder.io/sdk-svelte';
 
-/** @type {import('../$types').PageServerLoad} */
+// /** @type {import('../$types').PageServerLoad} */
```

Comment:
should we comment this part out? if you check their examples: https://kit.svelte.dev/docs/page-options#prerender-prerendering-server-routes

![Screenshot 2024-08-09 at 3 50 47 PM](https://github.com/user-attachments/assets/77aecf44-aba5-4e6d-97c6-e7b9010e6255)

Reply:
We can keep it back.